apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java

```diff
@@ -817,26 +817,24 @@ private Map.Entry<Long, Set<Column>> groupResults(Map.Entry<Long, Set<Column>> r
     private Publisher<Map.Entry<Long, Set<Column>>> mapCountAndColumns(Result result) {
         return result.map((row, rowMetadata) -> Map.entry(
                 row.get("count", Long.class),
```

Comment:
Minor: better re-use `mapCount` as well everywhere, like you'r doing for the `getColumns`, that way DRY. You just need to refactor it a bit.

Reply:
The problem is that `mapCount` only returns count, and we have places where we only return it, and others where we return columns as well. What I can do is reuse the `row.get("count", Long.class)`

Reply:
Yeah, I meant that. Anyway, it's too small might not be worthy encapsulating. Feel free to do whatever you like. It's approved.

Reply:
That's what I thought, but maybe extracting some of these mapping logic to a mapper class would be a good point. What do you think?

Reply:
Personally, I'd just push this. But feel free to do it provided that it doesn't overcomplicate things nor takes too long.

Reply:
It's a simple change, and this is not blocking anything.